integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -147,6 +149,41 @@ def get_unread_feedback_context(self) -> dict[str, QuerySet]:
             "unread_feedback": unread_feedback,
         }
 
+    def get_user_chat_stats(self) -> dict[str, QuerySet]:
+        r"""
+        Extend context by latest and total number of Frag Integreat chats.
+
+        :return: Dictionary containing the context for the Frag Integreat chats
+        """
+        total_chats = UserChat.objects.filter(region=self.request.region)
+        chats_last_month = UserChat.objects.filter(
+            region=self.request.region,
+            last_message_timestamp__gt=datetime.datetime.today()
+            - datetime.timedelta(days=30),
+        )
+        chats_previous_month = UserChat.objects.filter(
+            region=self.request.region,
+            last_message_timestamp__lt=datetime.datetime.today()
```

Comment:
```suggestion
            last_message_timestamp__lte=datetime.datetime.today()
```
Otherwise the 30th day in the past is not included, right - or am I missing something? :thinking: